libkbfs/bcache.go

```diff
@@ -85,10 +90,16 @@ func (b *BlockCacheStandard) Get(ptr BlockPointer) (Block, error) {
 		return b.cleanPermanent[ptr.ID]
 	}()
 	if block != nil {
-		return block, nil
+		return block, true, nil
```

Comment:
K. From my perspective, it's because a permanent entry can only be created if this client is performing a write. Since the client is writing, it knows what goes into it, including any potential directory entries or indirect blocks.